nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -978,3 +978,11 @@ To provide some examples:
 | `"foo", "bar"`       | `"baz", "bar"`  | no match           | match            | One compatible string matching is enough   |
 
 Note that this also allows writing overlays that explicitly apply to multiple boards.
+
+## NixOS Wiki {#sec-release-24.11-wiki}
+
+The official NixOS Wiki at [wiki.nixos.org](https://wiki.nixos.org/) was launched in April 2024, featuring
+content initially copied from the community wiki. The wiki enhances the official documentation, linking to
+existing resources and providing a categorization system for easy navigation, and is guided by a new "Manual
+of Style" — a contribution guide and enhanced templates. It offers a wealth of new information, including
+articles on applications, desktop environments, and a growing number of translations in multiple languages.
```

Comment:
I think this is unintentionally hard-wrapped?

Reply:
No, this is intentional. Just figured it was a very, very long line, and it looks nicer with the section above

It doesn't seem to mess with the render either
![image](https://github.com/user-attachments/assets/f854d9d6-c1fb-47a1-92f6-251727daced1)